src/utils/gui.cpp

```diff
@@ -30,7 +30,7 @@
  * Checks if there is at least one child that is visible
  */
 bool Utils::Gui::isOneTreeWidgetItemChildVisible(QTreeWidgetItem *item) {
-    for (int i = 0; i < item->childCount(); i++) {
+    for (int i = 0; i < item->childCount(); ++i) {
```

Comment:
This also does not the same than `i++` in this case!

Reply:
Really? :thinking: 

Okay done.

Reply:
Maybe I'm wrong in this case. I'm not 100% sure, but why are you changing those anyway? :grimacing: :grin:

Reply:
They are not so important. The compiler can optimize them but in general it is preferred to use the pre increment because its faster